framework/components/ecl-profile-topbars/ecl-profile-topbars.twig

```diff
@@ -1,31 +1,23 @@
-<div class="profile_topbar">
-
-    <div class="container">
-        <div class="section__group row">
-            <div class="col-md-2 profile_topbar__image">
-                <img src="http://placehold.it/160x160"/>
-            </div>
-
-            <div class="col-md-10 profile_topbar__column">
-                <div class="context-nav group-context-nav field-group-context_nav"><span
-                        class="context-nav__label">Department</span>
-                    <div class="context-nav__items">
-                        <span class="context-nav__item"><a href="#">Trade</a></span>
-                        <span class="context-nav__item"><a href="#">European Anti-Fraud Office</a></span>
-                    </div>
-                </div>
-
-                <button class="btn btn-collapse" data-target="#collapse" data-toggle="collapse">
-                    <span class="toggling-text">Show</span>
-                    <span class="icon icon--down ecl-u-color-primary"></span>
-                </button>
-            </div>
-        </div>
-    </div>
-
-    <div id="collapse" class="profile_topbar__expander">
-        <div class="container">
-            Expander content
-        </div>
+<div class="ecl-profile-topbar">
+  <div class="ecl-container">
+    <div class="ecl-row ecl-profile-topbar__wrapper">
+      <div class="ecl-col-md-2">
+        <img src="{{ profile.avatar }}"/>
```

Comment:
thanks, changed to `{% include '@ec-europa/ecl-images' with profile.image %}` :)